src/main/java/com/minecolonies/client/gui/WindowBuildTool.java

```diff
@@ -546,8 +529,37 @@ private void moveDownClicked(Button button)
     private void rotateRightClicked(Button button)
     {
         rotation = (rotation + ROTATE_RIGHT) % POSSIBLE_ROTATIONS;
-        RotationHelper.rotate(Settings.instance.getSchematicWorld(), EnumFacing.UP, true);
-        updatePosition();
+        updateRotation(rotation);
+    }
+
+    /**
+     * Updates the rotation of the structure depending on the input.
+     * @param rotation the rotation to be set.
+     */
+    private void updateRotation(final int rotation)
+    {
+        PlacementSettings settings = new PlacementSettings();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'settings' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)